Reqnroll.Tools.MsBuild.Generation/build/Reqnroll.Tools.MsBuild.Generation.targets

```diff
@@ -92,6 +92,7 @@
 
     <GenerateFeatureFileCodeBehindTask
       ProjectPath="$(MSBuildProjectFullPath)"
+      IntermediateOutputPath="$(BaseIntermediateOutputPath)"
```

Comment:
As far as I have seen the additional files are usually generated to `obj\Debug\net8.0`. This is where MS generates the `....AssemblyAttributes.cs` and `...GlobalUsings.g.cs` files as well and our MsTest/NUnit/etc plugins also generate the `MSTest.AssemblyHooks.cs` here. 
I have checked and this can be achieved by using the `$(IntermediateOutputPath)` instead of the `$(BaseIntermediateOutputPath)`. As far as I see nothing else needs to be changed in the code.

Reply:
done.